meta/main.yml

```
@@ -17,12 +17,9 @@ galaxy_info:
   min_ansible_version: "2.10"
   namespace: cisagov
   platforms:
-    # Amazon Linux 2023 does not (yet?) offer docker-compose or the
-    # Docker compose plugin:
-    # https://github.com/amazonlinux/amazon-linux-2023/issues/186
-    # - name: Amazon Linux 2023
-    #   versions:
-    #     - any
+    - name: Amazon Linux
+      versions:
+        - 2023
```

These are version strings so we must ensure they are not interpreted as numbers.

```suggestion
        - "2023"
```